Automattic/wp-calypso
# Jetpack pricing: add open-source section

### Changes proposed in this Pull Request

This PR adds a section mentioning open-source to the Jetpack pricing page.

Context: document linked in p1HpG7-j1R-p2

### Testing instructions

- Spin-up the pricing page in Cloud, using the live link below, or by running this branch locally
- Check that the open-source section looks like the capture below
- Do the same with Calypso blue

### Screenshots

_Cloud_
<img width="1091" alt="Screen Shot 2022-11-15 at 10 19 48 AM" src="https://user-images.githubusercontent.com/1620183/201956436-0d72c748-8d0a-479e-ab09-37613eae6a28.png">

_Calypso blue_
<img width="1022" alt="Screen Shot 2022-11-15 at 10 20 14 AM" src="https://user-images.githubusercontent.com/1620183/201956470-9f649aa3-26f7-4faa-a26b-073e25346c42.png">
